src/charts/line/Context.tsx

```diff
@@ -85,7 +85,7 @@ export function LineChartProvider({
   useAnimatedReaction(
     () => currentIndex.value,
     (x, prevX) => {
-      if (x !== -1 && x !== prevX && onCurrentIndexChange) {
+      if (x !== prevX && onCurrentIndexChange) {
```

Comment:
+1 to this. This change would allow `onCurrentIndexChange` to support detecting when the user ends interaction with the chart. Currently the `onActivated` and `onEnded` props don't work for me.

Reply:
Here's a workaround:

Create a handler that hooks into `LineChart.useChart()` to access `isActive`
```ts
const ChartInteractionHandler = ({
  onStart = () => {},
  onEnd = () => {},
}: {
  onStart?: () => void
  onEnd?: () => void
}) => {
  const { isActive } = LineChart.useChart()

  useAnimatedReaction(
    () => isActive.value,
    (active, wasActive) => {
      if (active && !wasActive) {
        runOnJS(onStart)()
      } else if (!active && wasActive) {
        runOnJS(onEnd)()
      }
    },
    [isActive],
  )

  return null
}
```
Then inside of the `<LineChart.Provider>` add:
```ts
<ChartInteractionHandler onStart={...} onEnd={...} />
```